nat-lab/tests/test_mesh_exit_through_peer.py

```diff
@@ -243,20 +243,21 @@ async def test_ipv6_exit_node(
         (alpha, beta) = api.default_config_two_nodes(
             alpha_ip_stack=IPStack.IPv4v6, beta_ip_stack=exit_ip_stack
         )
-        (connection_alpha, alpha_conn_tracker) = await exit_stack.enter_async_context(
+        (connection_alpha, _) = await exit_stack.enter_async_context(
             new_connection_with_conn_tracker(
                 alpha_connection_tag,
                 generate_connection_tracker_config(
                     alpha_connection_tag, derp_1_limits=ConnectionLimits(1, 1)
                 ),
             )
         )
-        (connection_beta, beta_conn_tracker) = await exit_stack.enter_async_context(
```

Comment:
What was the reason to remove conntrack checks?

Reply:
but conntracker here is enforcing connection count limits. Not specifically testing for STUN6. 

So the limits should be enforcable regardless of IP stack. Or am I missing something here?

Reply:
The `beta_conn_tracker` does not report `stun6` event, because it _sees_ mesh ip address (and not a real IPv6).